docs/providers/yandexcloud.rst

```diff
@@ -0,0 +1,8 @@
+yandexcloud
```

Comment:
For your information, this file is auto-generated in fact. I need to be more explicit about it.

Reply:
I used `./utils/generate_provider_doc.py` to generate it, it is okay.